docs/dev/release.md

```diff
@@ -30,42 +30,70 @@ All bundles/package manifests for Operators for operatorhub.io reside in:
 * `https://github.com/operator-framework/community-operators/tree/master/upstream-community-operators` - for public Operators from operatorhub.io
 * `https://github.com/operator-framework/community-operators/tree/master/community-operators` - for Operators from "internal" operatorhub that are synchronized with Openshift clusters
```

Comment:
probably repo name is different here too

Reply:
True, thanks 🙏